include/vcpkg/base/message-data.inc.h

```diff
@@ -330,6 +330,10 @@ DECLARE_MESSAGE(AVersionDatabaseEntry, (), "", "a version database entry")
 DECLARE_MESSAGE(AVersionObject, (), "", "a version object")
 DECLARE_MESSAGE(AVersionOfAnyType, (), "", "a version of any type")
 DECLARE_MESSAGE(AVersionConstraint, (), "", "a version constraint")
+DECLARE_MESSAGE(AzcopyFailedToPutBlob,
```

Comment:
I want to check if we already have a message that serves this purpose, I think we do... (no change requested from you now)

Reply:
The existing one I was thinking of is `msgProgramReturnedNonzeroExitCode` but that doesn't include the URL, so this is fine.